packages/@sanity/form-builder/src/inputs/BlockEditor-slate/conversion/slateHTMLDeserializer/helpers.js

```diff
@@ -21,20 +50,38 @@ function cleanUpWordDocument(html) {
     unwanted.parentNode.removeChild(unwanted)
   }
 
-  // Transform titles into H1s
-  const titleElments = document.evaluate(
-    "//p[@class='MsoTitle']",
+  // Transform mapped elements into what they should be mapped to
+  const mappedElements = document.evaluate(
+    `${mappedPaths.join('|')}`,
```

Comment:
No need to wrap this in a template string?

Reply:
Lol, no. There were once a need, but not anymore.